src/ContractInteract/UtilityToken.js

```diff
@@ -0,0 +1,346 @@
+/**
+ * @typedef {Object} UtilityTokenConfig
```

Comment:
`UtilityTokenConfig` should be `UtilityTokenSetupConfig`, to be consistent with the other setup configs.